lib/funky/funky_object.rb

```diff
@@ -0,0 +1,16 @@
+module Funky
+  class FunkyObject
+    extend DataParser
```

Comment:
@bruce-fs Since `DataParser` is only imported here, and since parsing the data is the whole goal of this class, could you delete the `DataParser` module and just have its code in this same file? Thanks.